commons/src/main/java/org/dhis2/commons/filters/FilterManager.java

```diff
@@ -698,9 +699,7 @@ private int getTotalFilterCounterForWorkingList(WorkingListScope scope) {
         int enrollmentStatusCount = scope.enrollmentStatusCount() != 0 ? 1 : 0;
         int eventStatusCount = scope.eventStatusCount() != 0 ? 1 : 0;
         int eventAssignedToMeCount = scope.assignCount() != 0 ? 1 : 0;
-        int total = eventDateCount + enrollmentDateCount + enrollmentStatusCount + eventStatusCount + eventAssignedToMeCount;
-        int workingListTotalFilters = total == 0 ? total : total + 1;
-        return workingListTotalFilters;
+        return eventDateCount + enrollmentDateCount + enrollmentStatusCount + eventStatusCount + eventAssignedToMeCount;
```

Comment:
Is there a way that we could add a test for this? to be sure we cover the expected result

Reply:
Of course!